cms/static/js/views/group_edit.js

```diff
@@ -1,7 +1,7 @@
 define([
```

Comment:
This is the editor for groups belonging to content experiment group configurations.